### PR TITLE
Fixed minor spelling mistakes in README.md

### PR DESCRIPTION
**Pull request by Wandmalfarbe**
**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

0

---
I just fixed two minor spelling mistakes :)

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ You can view [real-time stats](https://discourse.pi-hole.net/t/how-do-i-view-my-
 -   [Get LED alerts for each blocked ad](http://thetimmy.silvernight.org/pages/endisbutton/)
 -   [Pi-hole on Ubuntu 14.04 on VirtualBox](http://hbalagtas.blogspot.com/2016/02/adblocking-with-pi-hole-and-ubuntu-1404.html)
 -   [Docker Pi-hole container (x86 and ARM)](https://hub.docker.com/r/diginc/pi-hole/)
--   [Splunk: Pi-hole Visualizser](https://splunkbase.splunk.com/app/3023/)
+-   [Splunk: Pi-hole Visualiser](https://splunkbase.splunk.com/app/3023/)
```
Comment (PromoFaux) [inline README.md]: Depends where in the world you are based. I write in proper English too ;), but the project was made by our cousins over the pond who speak American English.

Comment (Wandmalfarbe) [inline README.md]: But with both an `s` and a `z` it was not correct. Right?

Comment (PromoFaux) [inline README.md]: Ha, I did not spot that...!

I'm happy with `s` :)

**Comment by PromoFaux** [comment]
Approved

**Comment by DL6ER** [comment]
Approved

**Pull request merged**